operator/controller/webhook/introute/sync.py

```diff
@@ -103,6 +129,35 @@ def _spring_cloud_k8s_config(parent) -> Optional[Mapping]:
         }
     }
 
+    return {
+        "name": "SPRING_APPLICATION_JSON",
+        "value": json.dumps(config),
+    }
+
+
+def _get_java_jdk_options(parent) -> Optional[Mapping[str, str]]:
+    tls_config = parent["spec"].get("tls")
+
+    if tls_config:
+        return {
+            "name": "JDK_JAVA_OPTIONS",
```

Comment:
Spring Boot added [SSL Bundles](https://spring.io/blog/2023/06/07/securing-spring-boot-applications-with-ssl/) in 3.1 to simplify setting up secure connections. It might be worth shifting over to that method.

Reply:
Will wait on more autoconfiguration support for `SSL Bundles` before adopting this change.